src/functions-and-arrays.js

```diff
@@ -1,18 +1,58 @@
 // Iteration #1: Find the maximum
-
+function maxOfTwoNumbers(a,b) {
+  if(a>b) {
+    return a;
+  }else if(b>a){
```

Comment:
Just an _else_ in this case is enough: 
`function maxOfTwoNumbers(a, b) {`
  `if(a > b) {`
    `return a;`
 ` } else {`
    `return b;`
 ` }`
`}`
`maxOfTwoNumbers(15, 30)`